sys/net/gnrc/link_layer/lorawan/Kconfig

```diff
@@ -16,6 +16,10 @@ menuconfig KCONFIG_USEMODULE_GNRC_LORAWAN
 
 if KCONFIG_USEMODULE_GNRC_LORAWAN
 
+config GNRC_LORAWAN_1_1
+    bool "Use LoraWAN version 1.1x"
+    default n
+
```

Comment:
In an offline discussion with @leandrolanzieri we realized GNRC LoRaWAN is still not modeled in the dependency resolution. Therefore, we cannot include it as pseudomodule from Kconfig.

You have two options here
1. Either try to model the dependency resolution of Kconfig
2. Remove this symbol from Kconfig